packages/manager/apps/veeam-enterprise/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@ovh-ux/manager-veeam-enterprise-app",
+  "version": "0.0.0",
```

Comment:
Is the expected behavior to use a regular commit type `feat` (without any breaking change I mean)?

Because it will result with a `0.1.0` version. Maybe you prefer `1.0.0`?

It's an open question.
Thanks!

Reply:
Seen together. `0.1.0` will be fine.